src/NodaTime/TimeZones/BclDateTimeZone.cs

```diff
@@ -193,12 +193,44 @@ internal BclAdjustmentRule(TimeZoneInfo zone, TimeZoneInfo.AdjustmentRule rule)
                 {
                     var daylightRecurrence = new ZoneRecurrence(zone.DaylightName, Savings, ConvertTransition(rule.DaylightTransitionStart), int.MinValue, int.MaxValue);
                     var standardRecurrence = new ZoneRecurrence(zone.StandardName, Offset.Zero, ConvertTransition(rule.DaylightTransitionEnd), int.MinValue, int.MaxValue);
-                    var recurringMap = new StandardDaylightAlternatingMap(StandardOffset, standardRecurrence, daylightRecurrence);
+                    IZoneIntervalMap recurringMap = new StandardDaylightAlternatingMap(StandardOffset, standardRecurrence, daylightRecurrence);
+                    // Fake 1 hour savings if the adjustment rule claims to be 0 savings.
```

Comment:
Good plan, will do. Completely agree that it's weird - definitely contrary to my normal instincts, but it feels like it just about crosses the line into being appropriate.